COPYRIGHT

```diff
@@ -1,6 +1,7 @@
 The code and documentation in this directory ("libcperciva") is distributed
 under the following terms:
 
+Copyright 2005-2020 Tarsnap Backup Inc.  All rights reserved.
```

Comment:
Make that 2011-2020 since Tarsnap Backup Inc. didn't exist before 2011.

Reply:
ok.  I picked 2005 up from the scrypt COPYRIGHT file, so I'll fix this in other projects as well.